pkgs/development/libraries/opencascade/opencascade-oce-vtk.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/IVtkVTK/IVtkVTK_ShapeData.cxx	2020-09-04 10:24:46.000000000 +0200
++++ b/src/IVtkVTK/IVtkVTK_ShapeData.cxx	2020-09-04 10:58:12.000000000 +0200
+@@ -104,7 +104,7 @@
+                                     const IVtk_PointId  thePointId2,
+                                     const IVtk_MeshType theMeshType)
+ {
+-  vtkIdType aPoints[2] = { thePointId1, thePointId2 };
++  vtkIdType aPoints[2] = { static_cast<vtkIdType>(thePointId1), static_cast<vtkIdType>(thePointId2) };
```

Comment:
I'm surprised that you need an explicit type casting here. What problem does this solve?

Reply:
Sorry, was writing this before reading your description. It would be a good idea to submit this patch for upstream review.

Reply:
Done. Though I don't know when it will be picked up.

Reply:
Can you use `fetchpatch` to fetch it from an url? Or at least provide a link to a ticket?

Reply:
Done.

Reply:
Thank you. This file is not needed then.